src/usr/local/www/system_certmanager.php

```diff
@@ -772,11 +794,31 @@ function list_csrs() {
 	}
 
 	$section->addInput(new Form_Select(
+		'keytype',
+		'*Key type',
+		$pconfig['keytype'],
+		array_combine($cert_keytypes, $cert_keytypes)
+	));
+
+	$group = new Form_Group($i == 0 ? '*Key length':'');
+        $group->addClass('rsakeys');
+	$group->add(new Form_Select(
 		'keylen',
-		'*Key length',
+		null,
 		$pconfig['keylen'],
 		array_combine($cert_keylens, $cert_keylens)
 	));
+	$section->add($group);
+
+	$group = new Form_Group($i == 0 ? '*EC key':'');
```

Comment:
"EC Key" is very user unfriendly. It should be spelled out ("Elliptic Curve Name")

Reply:
changed to Elliptic Curve Name